data/schema/schema_v1.json

```diff
@@ -533,16 +534,19 @@
                               "type": "integer"
                             },
                             "alias": {
-                              "type": "string"
+                              "type": "string",
+                              "description": "Allows another name for a variable to be used."
                             },
                             "repeats": {
-                              "type": "boolean"
+                              "type": "boolean",
+                              "description": "Value showing whether there are multiple answers for a question or not."
                             },
                             "max_value": {
                               "type": "object",
                               "properties": {
                                 "value": {
-                                  "type": "integer"
+                                  "type": "integer",
+                                  "description": "Maximum numeric value allowed in answer."
```

Comment:
- I think we can drop the 'in answer' bit. Same for `min_value`. 
- The `answer_id` also needs a description, something like "The id of an answer from which to obtain the max_value", and something similar for the `min_value.answer_id`.

Reply:
Done